pkgs/tools/misc/grub/pvgrub_image/configs/grub.cfg

```diff
@@ -0,0 +1,9 @@
+if search -s -f /boot/grub/grub.cfg ; then
+	echo "Reading (${root})/boot/grub/grub.cfg"
```

Comment:
The parentheses are here to match Grub's config file syntax. In fact, the file has been taken from [here](https://wiki.xen.org/wiki/PvGrub2). If you think that there is a good reason to remove them, then of course I can do it.